packages/docusaurus-plugin-content-pages/package.json

```diff
@@ -2,12 +2,16 @@
   "name": "@docusaurus/plugin-content-pages",
   "version": "2.0.0-alpha.25",
   "description": "Pages content plugin for Docusaurus",
-  "main": "src/index.js",
+  "main": "lib/index.js",
+  "scripts": {
+    "tsc": "tsc"
+  },
   "publishConfig": {
     "access": "public"
   },
   "license": "MIT",
   "dependencies": {
+    "@docusaurus/types": "^2.0.0-alpha.25",
```

Comment:
@endiliey should this be in dev or non-dev? ESLint was complaining that this should be in non-dev deps. Since it's used in `index.ts` I think it might be right 🤔

Reply:
by right should be dev bcs its only types. sometimes eslint can misunderstand

Reply:
Yeah I understand that but are there any implications about putting it in non-dev?

Reply:
user has to download this types package as well when they install preset-classic if its listed as non-dev

save few milliseconds of installation time